README.md

```diff
@@ -22,29 +24,53 @@ Refer to the Appendix on the [exposed user functions](https://github.com/privacy
 
 Refer to [Testing and Links](#testing-and-links) on detailed API documentations.
 
-
 ## Example: Fibonacci Circuit
+
 Run the following in command line:
+
 ```
 cargo run --example fibonacci
 ```
 
 This example demonstrates how to construct signals, step types, constraints, and witness generation in Chiquito. Best for first time Chiquito users.
 
 ## Example: MiMC7 Circuit
+
 TODO: annotate this code example
 
 This example demonstrates how to construct a lookup table and use external inputs as trace arguments in Chiquito, besides covering concepts in the Fibonacci example. MiMC7 is a zk-friendly hashing function.
 
 ## Example: zkEVM Bytecode Circuit
+
 https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/1348
 
 This example rewrites the zkEVM bytecode circuit using Chiquito and passes all original tests. It demonstrates how Chiquito can standardize and simplify larger scale circuits on the production level.
 
+## Python bindings
+
+### Install chiquito with pip
+
+```bash
+pip install chiquito
+```
+
+### Build from source
+
+In the root of this repo run
+
+```bash
+python -m venv .env
+source .env/bin/activate
+pip install maturin
+maturin develop
+```
```

Comment:
Thanks for adding this!

Are "install chiquito with pip" and "build from source" considered two different methods for installation or two steps of installation?

To confirm, after `pip install chiquito`, is `rust_chiquito` directly accessible from `chiquito.rust_chiquito` or requires `maturin develop`? My guess is the latter.

Reply:
It's either to install, pip package will be available for python user, `maturin develop` is only for who want to build from source/develop. 
Will make it clear